BridgeAppSDK/SBAActiveTask.swift

```diff
@@ -44,6 +44,7 @@ public enum SBAActiveTaskType {
     case walking
     case tremor
     case moodSurvey
+    case trailmaking
```

Comment:
I was figuring on including it here and sending Faraz a link to this PR so he could see what changes are required to implement, though that does remind me to add it to the sample app as well.